Assets/MRTK/SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs

```diff
@@ -168,6 +168,9 @@ protected override async void Start()
 
             await EnsureInputSystemValid();
 
+            // Setting the pointerName
+            PointerName = $"{Handedness}_{gameObject.name}";
+
             // We've been destroyed during the await.
             if (this == null)
```

Comment:
We may want to move this below the `if (this == null)` check. If `this == null`, then `gameObject` will likely be null too and `gameObject.name` will null ref

Reply:
The `this == null` check is meant to guard against the case where a hand is detected and lost within a single frame or a couple frames (either way, in fewer frames than it took for `await EnsureInputSystemValid();` to return). In that case, the pointer might be cleaned up and gone when we get here